internal/controllers/machine/machine_controller_status.go

```diff
@@ -405,7 +405,7 @@ func summarizeNodeV1Beta2Conditions(_ context.Context, node *corev1.Node) (metav
 			}
 		}
 		if condition == nil {
-			message += fmt.Sprintf("Node %s: condition not yet reported", conditionType) + "; "
+			messages = append(messages, fmt.Sprintf("* Node.%s: condition not yet reported", conditionType))
```

Comment:
```suggestion
			messages = append(messages, fmt.Sprintf("* Node.%s: Condition not yet reported", conditionType))
```

Wondering if that should be upper case for consistency with what we do in summary.go

Example:
```yaml
* B: Condition not yet reported
* !C: Condition not yet reported
```